recipe/meta.yaml

```diff
@@ -1,46 +1,35 @@
-{% set sha256 = "27a849d6205838fb6cc3c1c21cb9800707a661bb21c6ce7fb13e99eb1f8a0c46" %}
-{% set version = "1.57.0" %}
 {% set name = "googleapis-common-protos" %}
+{% set version = "1.57.0" %}
 
 package:
   name: {{ name|lower }}-feedstock
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 27a849d6205838fb6cc3c1c21cb9800707a661bb21c6ce7fb13e99eb1f8a0c46
 
 build:
-  number: 0
-  skip: true  # [win and vc<14]
+  noarch: python
+  number: 1
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
-    - pip >=18.1
-    - setuptools >=34.0.0
+    - python >=3.7
 
 outputs:
   - name: {{ name }}
     build:
-      script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose
-      skip: true  # [win and vc<14]
+      script: {{ PYTHON }} -m pip install . -vv
     requirements:
-      build:
-        - python                                 # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
       host:
-        - python
-        - pip >=18.1
-        - protobuf >=3.12.0
-        - setuptools >=34.0.0
+        - python >=3.7
+        - pip
       run:
-        - python
-        - protobuf >=3.12.0
+        - python >=3.7
+        - protobuf >=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
```

Comment:
Dependencies have changed and are causing `pip check` to fail in downstream feedstocks (https://github.com/conda-forge/apache-airflow-providers-google-feedstock/pull/20)